.github/dependabot.yml

```diff
@@ -54,3 +54,34 @@ updates:
     schedule:
       interval: "weekly"
     target-branch: "release/0.5"
+
+groups:
+  serde:
+    patterns:
+      - serde
+      - serde_*
+  tracing:
+    patterns:
+      - tracing
+      - tracing-*
+  opentelemetry:
+    patterns:
+      - opentelemetry
+      - opentelemetry-*
+      - tonic
+  trillium:
+    patterns:
+      - trillium
+      - trillium-*
+  kube:
+    patterns:
+      - kube
+      - k8s-openapi
+  postgres:
+    patterns:
+      - tokio-postgres
```

Comment:
Should we add a `[tokio, tokio-*]` group? 8 such crates show up in Cargo.lock (7, if we exclude `tokio-postgres` which is covered in the `postgres` group.

Reply:
I don't think so, we have only ever gotten Dependabot PRs for tokio out of that group. IIUC we don't get version updates because they are not direct dependencies, while we would still get security updates on them as transitive dependencies. The grouping feature only applies to version updates. (plus, the churn on tokio-stream and tokio-util is pretty low)